src/pages/postgraphile/usage-library.md

```diff
@@ -137,6 +137,24 @@ create function get_x_something() returns text as $$
 $$ language sql stable;
 ```
 
+Everything returned by pgSettings is applied to the current session with ```set_config($key, $value, true)```
+
+Internal Postgres settings (keys without periods) can be changed directly such as 'role':
```

Comment:
Consider changing this to something like:

You can use `pgSettings` to define variables that your Postgres functions/policies depend on, or to tweak internal Postgres settings. When adding variables for your own usage, the keys **must** contain either one or two period (`.`) characters, and the prefix (the bit before the first period) must not be used by any Postgres extension. Variables without periods will be interpreted as internal Postgres settings, such as `role`, and will be applied by Postgres. All settings are automatically reset when the transaction completes. Here's an example of switching the user into the Postgres 'visitor' role, and applying the application setting `jwt.claims.user_id`: